contracts/sol6/Dao/mock/KyberFeeHandlerWrapper.sol

```diff
@@ -63,21 +51,22 @@ contract KyberFeeHandlerWrapper is DaoOperator {
     /// @param endTokenIndex index of supportedTokens to start iterating from (exclusive)
     /// @param startKyberFeeHandlerIndex index of feeHandlerArray to start iterating from (inclusive)
     /// @param endKyberFeeHandlerIndex index of feeHandlerArray to start iterating from (exclusive)
+    /// @return amounts staker reward wei / twei amount claimed from each feeHandler
     function claimStakerReward(
         address staker,
         uint256 epoch,
         uint256 startTokenIndex,
         uint256 endTokenIndex,
         uint256 startKyberFeeHandlerIndex,
         uint256 endKyberFeeHandlerIndex
-    ) external returns(uint256 amountWei) {
+    ) external returns(uint256[] memory amounts) {
         if (
             startTokenIndex > endTokenIndex ||
             startKyberFeeHandlerIndex > endKyberFeeHandlerIndex ||
             supportedTokens.length == 0
         ) {
             // no need to do anything
-            return 0;
+            return amounts;
```

Comment:
if we return an uninitialised array
Calling contract can call amounts.length without reverting?

Reply:
Tested a simple example in remix, `amounts.length` will return 0.

Reply:
OK
thanks